soc/nordic/Kconfig

```diff
@@ -98,8 +98,7 @@ config NFCT_PINS_AS_GPIOS
 choice NRF_APPROTECT_HANDLING
 	bool "APPROTECT handling"
 	depends on SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET || \
-		   (SOC_NRF5340_CPUAPP && !TRUSTED_EXECUTION_NONSECURE) || \
```

Comment:
What about NRF54L ?

Reply:
I am not fully aware of it. Should it be added in here at this point?

Reply:
@vili-nordic: Given that the NRF54L is missing some support for approtect, it is better not to add it at this point.